src/main/java/at/fhhagenberg/esd/sqe/ws20/gui/ECCController.java

```diff
@@ -24,248 +20,290 @@
+    private final IntegerProperty weight = new SimpleIntegerProperty();
+    private final BooleanProperty anyElevatorSelected = new SimpleBooleanProperty();
+    private final Timer timer = new Timer();
+    private final StringProperty errorText = new SimpleStringProperty("");
+    @SuppressWarnings("unused")
+    @FXML
+    private ComboBox<String> cbElevator;
+    @SuppressWarnings("unused")
+    @FXML
+    private Label lCurFloor;
+    @SuppressWarnings("unused")
+    @FXML
+    private Label lNextTarFloor;
+    @SuppressWarnings("unused")
+    @FXML
+    private Label lDirection;
+    @SuppressWarnings("unused")
+    @FXML
+    private Label lSpeed;
+    @SuppressWarnings("unused")
+    @FXML
+    private Label lWeight;
+    @SuppressWarnings("unused")
+    @FXML
+    private ImageView ivDoorStateClosed;
+    @SuppressWarnings("unused")
+    @FXML
+    private ImageView ivDoorStateOpen;
+    @SuppressWarnings("unused")
+    @FXML
+    private ToggleButton tbtnOperationMode;
+    @SuppressWarnings("unused")
+    @FXML
+    private Label lTargetFloor;
+    @SuppressWarnings("unused")
+    @FXML
+    private ComboBox<String> cbTargetFloor;
+    @SuppressWarnings("unused")
+    @FXML
+    private Button btnGo;
+    @SuppressWarnings("unused")
+    @FXML
+    private TextArea taErrorLog;
+    @SuppressWarnings("unused")
+    @FXML
+    private GridPane gElevator;
+    @SuppressWarnings("unused")
+    @FXML
+    private GridPane gElevatorFloors;
+    @SuppressWarnings("unused")
+    @FXML
+    private Label lTopFloor;
+    @SuppressWarnings("unused")
+    @FXML
+    private Label lGroundFloor;
+    @SuppressWarnings("unused")
+    @FXML
+    private ImageView ivGElvDirUp;
+    @SuppressWarnings("unused")
+    @FXML
+    private ImageView ivGElvDirDown;
+    @SuppressWarnings("unused")
+    @FXML
+    private Label lElvCurFloor;
+    @SuppressWarnings("unused")
+    @FXML
+    private Rectangle recElevator;
+    @SuppressWarnings("unused")
+    @FXML
+    private Group groupElevator;
+    private FloorState[] floors;
+    private ReadOnlyIntegerProperty currentElevator;
+    private ReadOnlyIntegerProperty selectedFloor;
+    private IElevatorWrapper model;
+    private GeneralInformation info;
+
+    private static ImageView createFloorImageView(String path, ObservableValue<Boolean> visible) {
+        File file = new File(path);
+        Image image = new Image(file.toURI().toString());
+        ImageView iv = new ImageView();
+        iv.setPreserveRatio(true);
+        iv.setFitHeight(20);
+        iv.setImage(image);
+        iv.visibleProperty().bind(visible);
+
+        return iv;
+    }
+
+    public void setModel(IElevatorWrapper model) {
+        this.model = model;
+
+        //Init elevator floors
+        try {
+            info = model.queryGeneralInformation();
+        } catch (Exception e) {
+            errorText.concat(e.getMessage() + "\n");
+        }
+
```

Comment:
`info` is `null` if an `Exception` occurrs here

Reply:
Should be resolved with 3cf8c8e19736700d6cb75da0b6fa8bcb0ae1637c